velox/exec/tests/AssignUniqueIdTest.cpp

```diff
@@ -83,11 +83,13 @@ TEST_F(AssignUniqueIdTest, exceedRequestLimit) {
 }
 
 TEST_F(AssignUniqueIdTest, multiThread) {
-  vector_size_t batchSize = 1000;
-  auto input = {makeRowVector(
-      {makeFlatVector<int32_t>(batchSize, [](auto row) { return row; })})};
+  for (int i = 0; i < 3; i++) {
+    vector_size_t batchSize = 1000;
+    auto input = {makeRowVector(
+        {makeFlatVector<int32_t>(batchSize, [](auto row) { return row; })})};
 
-  auto plan = PlanBuilder().values(input, true).assignUniqueId().planNode();
+    auto plan = PlanBuilder().values(input, true).assignUniqueId().planNode();
 
-  verifyUniqueId(plan, input, 3);
+    verifyUniqueId(plan, input, 1000);
```

Comment:
1'000 threads seems impractical. Why not limit to something more realistic, e.g. 4 or 8?

Reply:
@mbasmanova I increase the value to 1000 to reproduce the issue easier. I can reduce it to 8